app-text/mupdf/patches/mupdf-1.19.1.patchset

```diff
@@ -0,0 +1,62 @@
+From 17ebe02ea6bfe59922c74de9b954958895515df7 Mon Sep 17 00:00:00 2001
+From: Ivan Holmes <ivan@ivanholmes.co.uk>
```

Comment:
Looking at previous patchset's it looks like (with some changes) a lot like the first one in line, maybe keep credits for the one that did the original patch?
https://github.com/haikuports/haikuports/blob/master/app-text/mupdf/patches/mupdf-1.6.patchset#L61

Reply:
So it does, I did it freshly myself though by copy-pasting the Linux section below and removing bits that didn't apply.
Quite happy to credit PulkoMandy though if you wish

Reply:
Would be nice yes :)

Reply:
Changed credit now